float32[] big_chicken_pos
float32[] small_chicken_pos
int32[] big_action
int32[] small_action
int32[] big_action_list
int32[] small_action_list
int32 big_cup
int32 small_cup
bool team
int32 status
int32 small_score
int32 big_score

